Funding rewrite by @katsinger

jpouellet opened the pull request:
This is an unreviewed cherry-pick of @katsinger's https://github.com/QubesOS/qubesos.github.io/pull/91 / https://github.com/QubesOS/qubesos.github.io/pull/106, with a few formatting nitpicks.

Git usage is cleaned up, rebased, and conflicts resolved, so now it's in a merge-able state if the content changes are approved.

/cc @mfc @andrewdavidwong 
see also: https://github.com/QubesOS/qubes-issues/issues/1701
xref: https://github.com/QubesOS/qubesos.github.io/pull/65 https://github.com/QubesOS/qubesos.github.io/pull/67 https://github.com/QubesOS/qubesos.github.io/pull/68

jpouellet commented on pages/donate.md:
```diff
@@ -1,80 +1,99 @@
 
-[funding]: /funding/
+[open-collective-faq]: https://opencollective.com/faq
+[verify address]: #
```
Was this supposed to be used and link someplace? Or should it be removed.

katsinger replied:
Line 87? It should be the linked to the anchor on that page: 

> ### <a name="verify-address"></a>How Do I Verify Your Bitcoin Address?

Thanks for catching that!

jpouellet replied:
In that case it should link to the anchor that gets auto-generated by the fact that it's a markdown header. Fixed.

katsinger commented:
@jpouellet, what is the reasoning behind putting nearly every sentence on a new line? Is that just part of the pull request? If those line breaks are to also be implemented as such on the website, I'd like the chance to convince whomever otherwise.

jpouellet commented:
They are not displayed like that in the rendered HTML, don't worry. Markdown requires a double line-break (new paragraph with space in-between) to force an actual line break.

The rationale for keeping one line per sentence in the source markdown is to ease reviewing diffs, as well as to ease translation workflows. More discussion can be found in https://github.com/QubesOS/qubes-issues/issues/2639

jpouellet commented:
I've fixed some more things and re-rebased to master.

jpouellet commented:
Note to reviewers: I'm not endorsing these changes, I'm just providing @katsinger help with git.

I personally have an easier time following the original format, but I'm neither a design nor marketing person, and I defer to those who are.

andrewdavidwong commented:
Thanks, @katsinger and @jpouellet! The Donations page looks great, but the Partners page looks a bit mangled.

By the way, is the intention to have this new Partners page replace the Funding page? It looks like it might be trying to duplicate some of the information from that page. (CC @rootkovska and @mfc)

katsinger commented:
@andrewdavidwong, here's the comment from my original pull request which should explain:

The content of the Donate and Partners pages has been rewritten from a communications standpoint to

    address the needs of the target audience
    provide a better information UX
    clarify the donation options
    simplify the donation process for potential donors.

The purpose of separate Funding and Partner pages was unclear and confusing. To remedy this, the information on the Funding page has been folded into the Donate and Partners pages and should therefore be removed.

rootkovska commented:
Yeah, there should be just one page, unifying the Funding/Partners, IMHO.

mfc commented:
this looks & reads great, thank you @katsinger for the rewrite!

andrewdavidwong commented:
Thanks all. This can be merged once the Partners page is fixed up.

jpouellet commented:
@katsinger you can start from a clean version of the changes as proposed in this PR by cloning the HEAD for this PR (`git clone --branch katsinger-rewrite-funding https://github.com/jpouellet/qubesos.github.io`).

andrewdavidwong commented:
Just a heads up: I've moved these pages to the qubes-doc repo as part of #2929. I didn't touch the content of the pages, though, so it shouldn't be a problem.

andrewdavidwong commented:
I'll close this PR so that @katsinger can open one against qubes-doc. Perhaps @jpouellet can assist with this if there are any questions.

katsinger commented:
@jpouellet, what do I do now exactly? I ran the command you gave me, so now I've got a new folder of the website on my desktop.

I'm also confused by the comment, "This can all be merged once the Partners page is fixed up," since the PR you submitted included my rewrite for that page as well. Or so I thought. So lost...

jpouellet commented:
> Thanks, @katsinger and @jpouellet! The Donations page looks great, but the Partners page looks a bit mangled.

If by mangled you mean how some html was being interpreted as markdown and leaking tags, then I've addressed that in https://github.com/jpouellet/qubesos.github.io/commit/76b6b3ffb4d048840f44032442c068e23cb61a44

As a separate issue, IMO the partner logos were unrecognizably small. There was some CSS [here](https://github.com/QubesOS/qubesos.github.io/blob/3551bcf/_sass/_pages.scss#L96-L121) which specified widths for columns, but these no longer corresponded to the same columns in the new table. I've addressed that in https://github.com/jpouellet/qubesos.github.io/commit/7b84853d717ce1a47ab895f81a99066ecbd111e4. I still think the old layout is easier on the eyes, but ¯\\\_(ツ)\_/¯

@andrewdavidwong Have I misinterpreted your meaning of mangled? If not I'll proceed with a separate PR updating just the CSS here and a PR against qubes-doc containing these changes. Would you prefer the partners changes separate from the funding/donations changes?

katsinger commented:
@jpouellet, thank you for clarifying; I had completely missed the "mangled" comment entirely! I will have a look at the page again now that I understand.

I made those edits so long ago, but I remember basically doing a guesswork job with the table and the logos in markdown. So I can understand where your IMHO comment is coming from!

jpouellet commented:
@katsinger You may find it helpful to [build and serve the site locally](https://github.com/QubesOS/qubesos.github.io#instructions) to view your changes before committing them. Doing so replaces all the guesswork with nearly-instant feedback.

andrewdavidwong commented:
We've moved these pages back to the main repo (see https://github.com/QubesOS/qubes-doc/commit/3db4fe4ed53bb332d4e8509ad24f80a021da906e#commitcomment-23216743), so I'm reopening this PR. Sorry for the flip-flop!

andrewdavidwong commented:
@jpouellet: You have the correct interpretation of "mangled."

jpouellet commented:
@andrewdavidwong so is this ready to be merged then? Or are you requesting further (perhaps stylistic?) changes? Next step is not clear to me.

andrewdavidwong commented:
@katsinger @jpouellet: Merged! Thank you both for all of your hard work!

katsinger commented:
Woohoo! Exciting to see it up. :)  @andrewdavidwong, thanks for your patience!  @jpouellet, thank you big time for your support!
